### ISSUE TITLE
Lose style of dbc table

### ISSUE THREAD

**Issue by bofish** [gabri-al/sarima_dashboard#1]
![image](https://github.com/gabri-al/sarima_dashboard/assets/28619249/6e388bdd-39e6-4f51-b89f-25398a0a7b5c)

I download the code from repo, but i found lose css style,
cloud you give some tips to solve this issue?

**Comment by bofish** [comment]
![image](https://github.com/gabri-al/sarima_dashboard/assets/28619249/e5f8a43b-e8d9-49e5-ab25-2b9a1a98ed3a)

I noticed that --bs-table-bg is white color when use bootstrap@5.3.2, so i rollback to version 5.2.3.

it's ok now

**Comment by gabri-al** [comment]
Hello,
Good catch! nice to know this. Probably we need to add some formatting to the whole table just in case..

**Comment by bofish** [comment]
yes, i try to add background color in custom_style.css, but it seems didn't work.
![image](https://github.com/gabri-al/sarima_dashboard/assets/28619249/66fd28c4-42ad-4e5e-8335-8c5bd3328267)
